atrackerv2/atracker_v2_integration_test.go

```diff
@@ -391,6 +416,28 @@ var _ = Describe(`AtrackerV2 Integration Tests`, func() {
 			Expect(target).ToNot(BeNil())
 		})
 
+		It(`ReplaceTarget(replaceTargetOptions *ReplaceTargetOptions) for event streams type of target`, func() {
+
+			eventstreamsEndpointPrototypeModel := &atrackerv2.EventstreamsEndpointPrototype{
+				TargetCRN: core.StringPtr("crn:v1:bluemix:public:messagehub:us-south:a/11111111111111111111111111111111:22222222-2222-2222-2222-222222222222::"),
+				Topic: core.StringPtr("my-test-topic"),
+				Brokers: []string{"kafka-x:9094"},
+				Password: core.StringPtr("xxxxxxxxxxxxx"),
+			}
+
+			replaceTargetOptions := &atrackerv2.ReplaceTargetOptions{
+				ID:             &targetIDLink2,
```

Comment:
```suggestion
				ID:             &targetIDLink3,
```

Reply:
Note that the cleanup code should also include a section to delete the target whose id is targetIDLink3